tls/s2n_config.h

```diff
@@ -50,6 +50,11 @@ struct s2n_config {
     s2n_cert_auth_type client_cert_auth_type;
     verify_cert_trust_chain_fn *verify_cert_chain_cb;
     void *verify_cert_context;
+
+    uint8_t mfl_code;
+
+    /* if this is FALSE, server will ignore client's Maximum Fragment Length request */
+    int enable_server_mfl;
```

Comment:
I like `accept_mfl` but not sure about `mfl_code_to_send` since it is "to_send" when u put in as a parameter at API but after it has been set on an extension and set in client & server config, `mfl_code_to_send` isn't really intuitive.